odl/operator/default_ops.py

```diff
@@ -877,9 +887,12 @@ def _call(self, x, out=None):
         rn(3).element([1.0, 2.0, 3.0])
         """
         if out is None:
-            return self.range.element(self.vector.copy())
+            if isinstance(self.__vector, float):
+                return self.range.element(self.__vector)
+            else:
+                return self.range.element(self.__vector.copy())
```

Comment:
Well if you are going to allow `float` you need to do more sweeping changes to naming etc. I wouldnt quite agree with "quick fixes" here.

Anyway, your fix would fail if the range was for example complex numbers, so you need to be abit more careful, also try to avoid `isinstance` in `_call` since it has some overhead. Perhaps settle this in the initializer? (`if isinstance(range, Field)`)